applications/optimization/integer_linear_programming/integer_linear_programming.ipynb

```diff
@@ -45,28 +45,6 @@
     "We go through the steps of solving the problem with the Classiq platform, using QAOA algorithm [[2](#QAOA)]. The solution is based on defining a pyomo model for the optimization problem we would like to solve."
```

Comment:
<p>CombinatorialProblem is a Python class, not a quantum object.</p><p>Use 'QAOA' (uper case), not 'qaoa'.</p><p>Use 'Pyomo' instead of 'pyomo'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>